architecture/index.adoc

```diff
@@ -0,0 +1,70 @@
+[id="architecture-overview"]
+= Architecture overview
+include::modules/common-attributes.adoc[]
+:context: architecture-overview
+
+toc::[]
+
+{product-title} is a cloud-based Kubernetes container platform.
+The foundation of {product-title} is based on Kubernetes and therefore shares the same technology.
+To learn more about {product-title} and Kubernetes, refer to the xref:../architecture/architecture.adoc#architecture[product architecture].
+
+[id="about-installation-and-updates"]
+== About installation and updates
+
+As a cluster administrator, you can use the {product-title} xref:../architecture/architecture-installation.adoc#architecture-installation[installation program] to install and deploy a cluster by using one of the following methods:
+
+** Installer-provisioned infrastructure (IPI)
```

Comment:
We actually don't allow IPI and UPI abbreviations for these in the docs, so suggest to remove them here